src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -135,8 +135,6 @@
     public String calculatingIslands = "%prefix% &7Calculating %amount% Players islands, Estimated time: %minutes% Minutes and %seconds% Seconds";
     public String calculatingFinished = "%prefix% &7Calculating Players islands Finished";
     public String calculationAlreadyInProcess = "%prefix% &7Calculating Players islands already in Process";
-    public String netherIslandsDisabled = "%prefix% &7Nether islands have been disabled";
```

Comment:
we don't need them if nether islands is disabled it should send player to nether world

Reply:
@Peaches_MLG do we really want it to behave that way?

Reply:
no I dont see that being useful

Reply:
so, what should i do, didn't understand

Reply:
Just block the portals and send this message

Reply:
So if netherIslands is disabled portals wont work on islands?? who wants it?
im still thinking we shouldn't block it <3